acquisitions/src/main/resources/thunderjet/mod-orders/reusable/create-order-lines.feature

```diff
@@ -0,0 +1,21 @@
+@ignore
+Feature: Create order lines
```

Comment:
There is already a feature to create an order line, it should be reused.

Reply:
> There is already a feature to create an order line, it should be reused.

Previously I tried to adapt it, I will give it another try now

Reply:
Damien, I've adopted your solution but removed the table UUIsD to let the mod-orders do the id generation on its own:

![image](https://github.com/folio-org/folio-integration-tests/assets/169707997/1ed03ef2-eb9c-4cf0-b94f-b5085f950da1)